ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/SFLOW_TESTS_LEAFS.yml

```diff
@@ -36,7 +36,7 @@ l3leaf:
 
 
   node_groups:
-    - group: sflow-tests-leaf-mlag
+    - group: SFLOW_TESTS_LEAF_MLAG
```

Comment:
this group should not be changed. Only the file names and the inventory.yml

Reply:
fixed